app/models/manageiq/providers/embedded_terraform/automation_manager/vsphere_credential.rb

```diff
@@ -0,0 +1,58 @@
+class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::VsphereCredential < ManageIQ::Providers::EmbeddedTerraform::AutomationManager::CloudCredential
```

Comment:
@agrare thoughts on the casing on `Vsphere`?  I see both kinds, 🤣 

`VSphereAutomation` gem used in the vmware provider and `VsphereProvisioning`, `VsphereReplication`, etc. in intersight client, oci.

Reply:
Technically the name is `vSphere` so if we are capitalizing the first letter `VSphere` would be most accurate, but honestly I think `Vsphere` is fine here

Reply:
Though I think "wrong answers only" for vMwAre VsPhere would be pretty funny

Reply:
nerd snipe Adam right before he has to start a call.  🏆 

Oops.  haha, I forgot it's actually vSphere.  Ok, so Vsphere is fine.

Reply:
Actually it was https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/16#discussion_r1576298959 that did it :D

Reply:
I knew if I did it halfway, you'd finish it